62_CAD/fragment_shader.hlsl

```diff
@@ -557,13 +572,36 @@ float4 main(PSInput input) : SV_TARGET
 
             const float dist = sqrt(closestDistanceSquared);
             localAlpha = 1.0f - smoothstep(0.0, globals.antiAliasingFactor, dist);
+            
+        }
+
+        LineStyle style = lineStyles[mainObjects[currentMainObjectIdx].styleIdx];
+        uint32_t textureId = asuint(style.screenSpaceLineWidth);
+        if (textureId != InvalidTextureIdx)
+        {
+            const float screenSpaceSize = 8.0;
```

Comment:
name your variables better, HatchFillMSDFScreenSpaceSize and move to common.hlsl

Reply:
Done